web/source/osk/visualKeyboard.ts

```diff
@@ -73,6 +69,37 @@ namespace com.keyman.osk {
       this._layerId = value;
     }
 
+    get currentLayer(): OSKLayer {
+      if(this.layerGroup && this.layerId) {
+        return this.layerGroup.layers[this.layerId];
+      }
+    }
+
+    // Special keys (for the currently-visible layer)
+    get lgKey(): KeyElement { // currently, must be visible for the touch language menu.
+      if(this.currentLayer && this.currentLayer.globeKey) {
+        return this.currentLayer.globeKey.btn;
+      } else {
+        return null;
+      }
```

Comment:
Should be able to do this, no?

```suggestion
      return this.currentLayer?.globeKey?.btn;
```

and ditto for `hkKey()`, `spaceBar()`?